plugins/inventory/k8s.py

```diff
@@ -14,7 +14,7 @@
     short_description: Kubernetes (K8s) inventory source
 
     description:
-      - Fetch containers and services for one or more clusters
+      - Fetch containers in pods for one or more clusters
```

Comment:
Is this a change in functionality or is this a correction to the documentation?

Reply:
That's a change in functionality; this PR would limit the scope of the inventory plugin to something that's more practical in real-world use, but because it changes the behavior and removes a feature, that should probably be put into a 2.x.